### ISSUE TITLE
2 suggestions for librelogin

### ISSUE THREAD

**Issue by pasieniec2** [kyngs/LibreLogin#294]
1. simple captcha e.g. /register {password} {password} {captcha}
2. Fix the problem when someone uses an online player's nickname. **I have an additional plugin that fixes this librelog error, but it's better to have everything in one** and not that I have to have an additional plugin to prevent account hacking

**Comment by pasieniec2** [comment]
Bump

Please updatee

**Comment by pasieniec2** [comment]
private plugin name: login fix

**Comment by pasieniec2** [comment]
Suggestion 3
/Librelogin user {player} acceptalts

**Comment by kyngs** [comment]
simple captcha e.g. /register {password} {password} {captcha}

Not planned, might add an API option for that later.

> Fix the problem when someone uses an online player's nickname. I have an additional plugin that fixes this librelog error, but it's better to have everything in one and not that I have to have an additional plugin to prevent account hacking

Dupe of #296

**Comment by pasieniec2** [comment]
what about proposition 3? as for 296, it seems to me that this post was earlier, so that one is a copy of this one, and this is a serious error. thanks for replying

> simple captcha e.g. /register {password} {password} {captcha}
> 
> Not planned, might add an API option for that later.
> 
> > Fix the problem when someone uses an online player's nickname. I have an additional plugin that fixes this librelog error, but it's better to have everything in one and not that I have to have an additional plugin to prevent account hacking
> 
> Dupe of #296

**Comment by kyngs** [comment]
Yes, this one was made before 296, but I've decided to track the progress on that there.
As for proposition 3, I'm sorry, I forgot about that message. I don't really know the proper way to implement this bypass. Do we want the bypass name-based, or ip-based? If the former, we can just user a permission for that.

**Comment by pasieniec2** [comment]
> Tak, ten powstał przed 296 rokiem, ale postanowiłem śledzić postępy w tym zakresie. Jeśli chodzi o propozycję 3, przepraszam, zapomniałem o tej wiadomości. Naprawdę nie znam właściwego sposobu wdrożenia tej obwodnicy. Czy chcemy, aby obwodnica była oparta na nazwie lub ip? Jeśli to pierwsze, możemy po prostu użyć do tego pozwolenia.

When it comes to permissions, it's more about IP because there's no way to work with nicknames because it limits accounts per IP, not per nickname. I found a simple way to do it, just enter /librelogin user cracked {nickname we want} to have as a new account
